src/store/modules/wallet.js

```diff
@@ -230,17 +230,14 @@ const actions = {
     try {
       if (state.signer && !getters.walletIsMatchedSession) {
         await dispatch('signLogin');
+      } else if (getters.walletHasLoggedIn) {
```

Comment:
The else relationship seems not intuitive, i.e. what does `!(state.signer && !getters.walletIsMatchedSession)` mean?
Maybe should name the if condition above to a meaningful name

Reply:
Updated

Reply:
Is it possible for `!state.signer & getters.walletIsMatchedSession` to exist?

Reply:
Updated with flipped conditions